### DISCUSSION TITLE
Frequencies for inversions of small events?

### DISCUSSION THREAD

**Discussion by Donavin97** [nikosT/Gisola#11] [Usage Questions]
Hi @nikosT 
I have a network of seismometers with 20 and 40 BH channels, as well as 100 HH channels.
My question is, what frequencies can I use to invert for events from magnitude 2.5 to 3.5 or so? Thanks

**Comment by nikosT** [comment]
Hi @Donavin97, I think also this question should be answered by @esokos.

**Comment by esokos** [comment]
Hello @Donavin97 ,
what is the 20,40,100 ..?? sampling rates or response frequency cutoff.?

if you want to invert events in this magnitude range the first thing you should consider is the epicentral distance ... you need enough stations close to epicenter (<20km i would say depends on level of noise in stations) and after that you may experiment with frequency bands i would say around 0.5Hz but needs some investigation.

 We have a paper here https://link.springer.com/book/10.1007/978-3-319-77359-9 where we discuss relevant situation e.g. How to choose the inverted frequency band?. 

have a look.

**Comment by Donavin97** [comment]
Thanks @esokos for clarifying.
The 20 40 and 100 are sample rates for instruments I have access too
